xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -1115,8 +1121,11 @@ int CGUIDialogVideoInfo::ManageVideoItem(const std::shared_ptr<CFileItem>& item)
     }
   }
 
-  if (type != MediaTypeSeason)
+  if (type != MediaTypeSeason && (item->m_bIsFolder || !item->IsVideoAssetNav()))
```

Comment:
I have spotted the term `(type == MediaTypeMovie && (item->m_bIsFolder || !item->IsVideoAssetNav()))` at several places in this commit. Is this something that could be  factored out to a new `CFileItem` method?

Reply:
Yes, good idea. I can remove IsVideoAssetNav() actually. At this point we only care about detecting the real files, excluding the "movies that are folders".
Also fixes a little glitch from previous PR in the process.